lago/export.py

```diff
@@ -145,7 +148,8 @@ def write_lago_metadata(self):
 
 class TemplateExportManager(DiskExportManager):
     """
-    TemplateExportManager is responsible exporting images of type template.
+    :class:TemplateExportManager is responsible exporting images of type
```

Comment:
same about the :class:, though might be wrong better check how it rendered them.
(side note - it is a bit confusing as we are using the google style of code, (i.e. Args:/Returns: instead of :class: etc), but for sphinx to render properly you need the :thing:).